ratpack-core/src/main/java/ratpack/exec/Operation.java

```diff
@@ -107,6 +107,18 @@ default Operation next(Block operation) {
     return next(Operation.of(operation));
   }
 
+  /**
+   * Executes the given block as an operation, on a blocking thread.
+   * <p>
+   *
+   * @param operation a block of code to be executed, on a blocking thread
+   * @return an operation
+   * @since 1.4
+   */
+  default Operation blockingNext(Block operation) {
+    return next(Blocking.op(operation));
+  }
+
```

Comment:
Should we also have `blockingNext(Operation)` since `next` can take `Block` or `Operation`?